.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,25 +1,24 @@
-<Include a few sentences describing the overall goals for this Pull Request. Please do not remove the checklist below, pull requests missing it will be ignored.>
 
-## Checklist
+<!--Include a few sentences describing the overall goals for this Pull Request-->
+  
+<!-- Please help our volunteers reviewing this PR by completing the following items. 
+Ask in a comment if you have troubles with any of them. -->
 
-> Reviewing is a process done by project maintainers, mostly on a volunteer basis. We try to keep the overhead as small as possible and appreciate if you help us to do so by completing the following items. Feel free to ask in a comment if you have troubles with any of them.
+# Checklist
 
-**Submitting the PR does not require you to check all items, but by the time it gets merged, they should be either satisfied or not applicable.**
+- [ ] I have read the [contribution guidelines](https://github.com/geoserver/geoserver/blob/main/CONTRIBUTING.md).
+- [ ] I have sent a [Contribution Licence Agreement](https://docs.geoserver.org/latest/en/developer/policies/committing.html) (not required for small changes, e.g., fixing typos in documentation).
+- [ ] First PR targets the `main` branch (backports managed later; ignore for branch specific issues).
+- [ ] All the build checks are green ([see automated QA checks](https://docs.geoserver.org/latest/en/developer/qa-guide/index.html)).
 
+For core and extension modules:
 
-For all pull requests:
+- [ ] New unit tests have been added covering the changes.
+- [ ] [Documentation](https://github.com/geoserver/geoserver/tree/main/doc/en) has been updated (if change is visible to end users).
+- [ ] The [REST API docs](https://github.com/geoserver/geoserver/tree/main/doc/en/api/1.0.0) have been updated (when changing configuration objects or the REST controllers).
+- [ ] There is an issue in the [GeoServer Jira](https://osgeo-org.atlassian.net/projects/GEOS) (except for changes not visible to end users). 
```

Comment:
```suggestion
- [ ] There is an issue in the [GeoServer Jira](https://osgeo-org.atlassian.net/GEOS/issues/) (except for changes that do not affect administrators or end users in any way). 
```

Reply:
The link is actually a 404. You probably meant https://osgeo-org.atlassian.net/jira/software/c/projects/GEOS/issues/

Reply:
I'm adding it in a commit to capture the wording change and then fix the link